Vic2ToHoI4/Source/Mappers/FactionName/FactionNameMapperFactory.h

```diff
@@ -0,0 +1,31 @@
+#ifndef FACTION_NAME_MAPPER_FACTORY_H
```

Comment:
Add tests for this file. The other mapper shave examples. You'll need to add a test file to the project and set it to copy to the proper directory.